app/src/main/java/de/danoeh/antennapod/fragment/ExternalPlayerFragment.java

```diff
@@ -80,50 +79,10 @@ private PlaybackController setupPlaybackController() {
         return new PlaybackController(getActivity(), true) {
 
             @Override
-            public void setupGUI() {
-            }
-
-            @Override
-            public void onPositionObserverUpdate() {
-                ExternalPlayerFragment.this.onPositionObserverUpdate();
```

Comment:
Removing this line breaks the progress bar in the external player (it won't ever update)